libraries/rush-lib/src/cli/parsing/SelectionParameterSet.ts

```diff
@@ -260,62 +260,6 @@ export class SelectionParameterSet {
     return selection;
   }
 
-  /**
-   * Represents the selection as `--filter` parameters to pnpm.
-   *
-   * @remarks
-   * This is a separate from the selection to allow the filters to be represented more concisely.
-   *
-   * @see https://pnpm.js.org/en/filtering
-   */
-  public async getPnpmFilterArgumentsAsync(terminal: ITerminal): Promise<string[]> {
-    const args: string[] = [];
-
-    // Include exactly these projects (--only)
-    for (const project of await this._evaluateProjectParameterAsync(this._onlyProject, terminal)) {
-      args.push('--filter', project.packageName);
-    }
-
-    // Include all projects that depend on these projects, and all dependencies thereof
-    const fromProjects: Set<RushConfigurationProject> = Selection.union(
-      // --from
-      await this._evaluateProjectParameterAsync(this._fromProject, terminal)
-    );
-
-    // All specified projects and all projects that they depend on
-    for (const project of Selection.union(
-      // --to
-      await this._evaluateProjectParameterAsync(this._toProject, terminal),
-      // --from / --from-version-policy
-      Selection.expandAllConsumers(fromProjects)
-    )) {
-      args.push('--filter', `${project.packageName}...`);
-    }
-
-    // --to-except
-    // All projects that the project directly or indirectly declares as a dependency
-    for (const project of await this._evaluateProjectParameterAsync(this._toExceptProject, terminal)) {
-      args.push('--filter', `${project.packageName}^...`);
-    }
-
-    // --impacted-by
-    // The project and all projects directly or indirectly declare it as a dependency
-    for (const project of await this._evaluateProjectParameterAsync(this._impactedByProject, terminal)) {
-      args.push('--filter', `...${project.packageName}`);
-    }
-
-    // --impacted-by-except
-    // All projects that directly or indirectly declare the specified project as a dependency
-    for (const project of await this._evaluateProjectParameterAsync(
-      this._impactedByExceptProject,
-      terminal
-    )) {
-      args.push('--filter', `...^${project.packageName}`);
-    }
-
-    return args;
-  }
```

Comment:
@william2958 I can't find any callers of this code. It looks like a fragment left over from the split workspace PR?

Reply:
This algorithm was actually pretty useful. @D4N14L and I were talking about this earlier today and concluded that it'd probably be a good idea to pull back the pnpm shorthands instead of always enumerating all of the projects that are getting installed.